packages/eslint-config-nextjs-ts/.eslintrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I can not reference the deleted file, but `package-diagram.jpg` is not needed since we use mermaid to generate graphs.